src/Socket.php

```diff
@@ -294,20 +298,26 @@ public static function dropdownWiringSide($name, $options = [])
             }
         }
 
-        return Dropdown::showFromArray($name, self::getSides(), $params);
+        return Dropdown::showFromArray($name, self::getSides($full), $params);
     }
 
 
     /**
      * Get sides
      * @return array Array of types
      **/
-    public static function getSides()
+    public static function getSides($full = false)
```

Comment:
```suggestion
    public static function getSides(bool $full = false)
```
Also, please add param in PHPDoc.